lib/improver/tests/cube_combiner/test_combine_cli.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2019 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Tests for the combine CLI"""
+
+import pytest as pt
+from improver.cli import combine
+from improver.tests import kgo, cli
```

Comment:
On the namespacing: `acctest`?

Reply:
I've changed this to use the name acceptance or abbreviated to "acc".  The code that was in `cli` under `improver.tests` goes away completely now that we're not testing usage and help messages.

Reply:
> Also, I wonder if it would it be worth to change the file naming pattern `test_xxx_cli.py` -> `test_cli_xxx.py` so it's like module namespacing for CLIs?

Thinking about again, we should instead rename and **move** this file `cube_combiner/test_combine_cli.py` -> `cli/test_combine.py` to follow module layout better. CLI tests don't belong with unit tests. Oh well, too late now but we should fix this in the follow on work.

Reply:
OK, I'll do this rename/move in the next CLI test related pull request.